pandas/tests/io/test_clipboard.py

```diff
@@ -69,18 +72,36 @@ def teardown_class(cls):
     def check_round_trip_frame(self, data_type, excel=None, sep=None,
                                encoding=None):
         data = self.data[data_type]
-        data.to_clipboard(excel=excel, sep=sep, encoding=encoding)
-        if sep is not None:
-            result = read_clipboard(sep=sep, index_col=0, encoding=encoding)
+        if excel in [None, True] and sep is not None and len(sep) > 1:
```

Comment:
Admittedly not overly familiar with this code but having a hard time making sense of it - is this possible to parametrize? We typically avoid a lot of conditionals and loops in test scenarios

Reply:
Do you mean like this one? https://github.com/pandas-dev/pandas/blob/e033c0616158d3ba974456b4f84810492936b1fe/pandas/tests/io/test_common.py#L183-L188

Reply:
Yes that's one example. There's plenty of them in the tests so feel free to poke around